Content.Shared/_CM14/Scoping/SharedScopeSystem.User.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Robust.Shared.Containers;
+using Robust.Shared.Player;
+
+namespace Content.Shared._CM14.Scoping;
+
+public abstract partial class SharedScopeSystem
+{
+    public void InitializeUser()
+    {
+        SubscribeLocalEvent<ScopeUserComponent, MoveEvent>(OnMove);
+        SubscribeLocalEvent<ScopeUserComponent, EntParentChangedMessage>(OnParentChanged);
+        SubscribeLocalEvent<ScopeUserComponent, ContainerGettingInsertedAttemptEvent>(OnInsertAttempt);
+        SubscribeLocalEvent<ScopeUserComponent, EntityTerminatingEvent>(OnEntityTerminating);
+
+        SubscribeLocalEvent<ScopeUserComponent, PlayerDetachedEvent>(OnPlayerDetached);
+    }
+
+    private void OnMove(EntityUid uid, ScopeUserComponent component, ref MoveEvent args)
+    {
+        if (!TryComp<ScopeComponent>(component.ScopingItem, out var scopeComponent))
+            return;
+
+        if (!_transformSystem.InRange(scopeComponent.LastScopedAt, args.NewPosition, 1.5f))
+            UserStopScoping(uid, component);
+    }
```

Comment:
`MoveEvent` is a very expensive event to subscribe to since it will trigger every single time an entity move and make physics slower to tick in general.
If it works, use what SharedWatchXenoSystem uses instead, `MoveInputEvent`.

Reply:
Actually won't anchor the user, as it might have an unintended consequence of players using their scopes to avoid being dragged off - will think of a different solution, for now will just unscope if being dragged.